providers/dns/webnames/webnames.toml

```diff
@@ -0,0 +1,31 @@
+Name = "Webnames"
+Description = ''''''
+URL = "https://www.webnames.ru/"
+Code = "webnames"
+Since = "v4.14.2"
+
+Example = '''
+WEBNAMES_API_KEY=xxxxxx \
+lego --email you@example.com --dns webnames --domains my.example.org run
+'''
+
+Additional = '''
+## API key 
+
+Для получения ключа необходимо сменить DNS-сервера на *.nameself.com: Личный кабинет / Мои домены и услуги / Выбрать необходимый домен / DNS-серверы
+
+Ключ API можно будет найти: Личный кабинет / Мои домены и услуги / Выбрать необходимый домен / Управление зоной / Настройка acme.sh или certbot
```

Comment:
you should write the doc in English.

Reply:
I can translate but this site does not have built-in English. Maybe it would be better to add a description in two languages?

Reply:
Only English